packages/core/src/interpreter.ts

```diff
@@ -475,6 +470,14 @@ export class Interpreter<
     };
   }
 
+  public getPersistedState(): PersistedFrom<TBehavior> | undefined {
+    if (!this._state) {
+      return undefined;
+    }
+
+    return this.behavior.getPersistedState?.(this._state);
```

Comment:
It feels like this could be read like this:
```suggestion
    return this.behavior.getPersistedState?.(this._state || this._getInitialState());
```

Any reason why we don't grab the persisted state for not-started behaviors?